src/settings/MappingProfiles/MappingProfilesFormContainer/MappingProfilesFormContainer.js

```diff
@@ -53,6 +54,20 @@ export const MappingProfilesFormContainer = props => {
         .find(transformation => selectedTransformation.fieldId === transformation.fieldId),
     ),
   );
+  const [disabledRecordType, setDisabledRecordType] = useState(null);
+  const [initiallyDisabledRecord] = useState(() => {
+    const disabledFields = {};
+
+    if (initialValues.recordTypes) {
```

Comment:
Please adjust the code so we won't need that check at all. By the way, can we move this code to the FolioRecordTypeField component at all?

Reply:
I don't think than it can be moved, since not only `FolioRecordTypeField` need this data, but also Transformation Modal as well